src/Components/Form/ArrayTextInput/ArrayTextInput.jsx

```diff
@@ -90,6 +91,7 @@ const ArrayTextInput = ({
                 });
               }}
               value={elem.value}
+              maxLength={maxLength}
```

Comment:
sholdnt we insert custom inputProps like this?
```suggestion
              {...inputProps}
```